buildpipeline/Core-Setup-RHEL7-x64.json

```diff
@@ -197,13 +219,15 @@
       "cloneUrl": "https://github.com/dotnet/core-setup.git",
       "refsUrl": "https://api.github.com/repos/dotnet/core-setup/git/refs",
       "gitLfsSupport": "false",
-      "fetchDepth": "0"
+      "skipSyncSource": "false",
+      "fetchDepth": "0",
+      "cleanOptions": "0"
     },
     "id": "https://github.com/dotnet/core-setup.git",
     "type": "GitHub",
     "name": "dotnet/core-setup",
     "url": "https://github.com/dotnet/core-setup.git",
-    "defaultBranch": "release/build_defs_1.1.0",
+    "defaultBranch": "BuildConfigurations",
```

Comment:
Corresponding file in 1.0.0 look good.  So this one is due to my carelessness 👎  Will fix up.